### PR TITLE
Availability check with Receptor client

### PR DESCRIPTION
**Pull request by slemrmartin**
Implementation of "receptor_satellite:health_check" receptor directive.
Also implemented new receptor feature, when responses in kafka are paired by "in_response_to" and each request have 1+ message of "response" type and last message of "eof" type.

Health check has exactly 1 message of type "response"

Receptor client prepared to extraction

### CODE REVIEW

**Comment by slemrmartin** [comment]
@agrare please :eyes: 
wip == missing specs, but code is ready

**Comment by agrare** [comment]
:+1: LGTM

File: lib/topological_inventory/satellite/receptor/client/response_worker.rb [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+          end
+
+          sleep(10)
+        end
+      end
+
+      def queue_opts
+        {
+          # :max_bytes   => 50_000,
+          # :persist_ref => "topological-inventory-receptor-responses", # groups subscribers (message is consumed by only one of the subscribers)
```
Comment (slemrmartin) [inline lib/topological_inventory/satellite/receptor/client/response_worker.rb]: @agrare do you know if this is correct settings?
1) `max_bytes`: If there is no `max_bytes`, then it's unlimited on this side?
2) `persist_ref`
Usually we need to read the message by exactly one subscriber, but there we need that all subscribers (if operations worker will be scaled) receives the message from kafka. 
We're using options like `group_ref` etc. in other repos  but I didn't find it in manageiq-messaging

Comment (agrare) [reply lib/topological_inventory/satellite/receptor/client/response_worker.rb]: @slemrmartin there might be some kafka default max_bytes but I'm not sure exactly, we'd need to check the ruby-kafka gem or the kafka broker docs specifically
If you don't pass a persist_ref then all subscribers in the group will be delivered the message

**Comment by slemrmartin** [comment]
@agrare tested, ready

**Pull request merged**